org.lflang.diagram/src/org/lflang/diagram/synthesis/styles/LinguaFrancaStyleExtensions.java

```diff
@@ -67,7 +67,10 @@
 @ViewSynthesisShared
 public class LinguaFrancaStyleExtensions extends AbstractSynthesisExtensions {
     
-    private static final Property<Colors> LABEL_PARENT_BACKGROUND = new Property<>(
+    /** 
+     * INTERNAL property to communicate a node's background color. 
```

Comment:
If internal then why `public`?

Reply:
I needed to change them to public in order to reference them in the blacklisting in SynthesisRegistration.java